build-tools/packages/build-cli/src/library/codeCoverageDangerFile.cts

```diff
@@ -0,0 +1,115 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Handle weirdness with Danger import.  The current module setup prevents us
+// from using this file directly, and the js transpilation renames the danger
+// import which prevents danger from removing it before evaluation (because it
+// actually puts its exports in the global namespace at that time)
+declare function fail(message: string, file?: string, line?: number): void;
+
+declare const danger: {
```

Comment:
This is just our hardcoded version of danger's API, right? For whatever reason we don't have its types? If so, can we add a comment to clarify that?

Reply:
Removed danger.